security/pkg/nodeagent/sds/sdsservice.go

```diff
@@ -316,6 +310,14 @@ func pushSDS(con *sdsConnection) error {
 		return err
 	}
 
+	if con.secret.RootCert != nil {
+		log.Infof("SDS: push root cert from node agent to proxy: %q\n", con.proxyID)
+		log.Debugf("SDS: push root cert %+v to proxy: %q\n", string(con.secret.RootCert), con.proxyID)
+	} else {
+		log.Infof("SDS: push key/cert pair from node agent to proxy: %q\n", con.proxyID)
```

Comment:
Maybe remove "key/" here to be consistent with the debugging log.

Reply:
I think it's ok(since we omit private key in debug logs), while push both actually.